cmp/form/dropdown/BaseDropdownField.js

```diff
@@ -25,7 +25,9 @@ export class BaseDropdownField extends HoistField {
 
     static defaultProps = {
         placeholder: 'Select'
-    }
+    };
+
+    commitOnChange = true;
```

Comment:
seems weird that we need to write this prop directly on the component, and override it there.  Is there no better way to set a default prop for these hoist fields to be different from the inherited value?